vultr-hook.sh

```diff
@@ -32,11 +32,15 @@ deploy_challenge() {
 
     echo " + ${SCRIPTNAME}: Creating entry $ENTRY in $TOP"
 
-    curl -s -H "API-Key: ${APIKEY}" 'https://api.vultr.com/v1/dns/create_record' \
-        --data "domain=${TOP}" \
-        --data "name=${ENTRY}" \
-        --data "type=TXT" \
-        --data "data=\"${TOKEN_VALUE}\""
+    curl -s -H "Authorization: Bearer ${APIKEY}" -H "Content-Type: application/json" \
+        -X POST "https://api.vultr.com/v2/domains/${TOP}/records" \
+        --data "{
+            \"name\" : \"${ENTRY}\",
+            \"type\" : \"TXT\",
+            \"data\" : \"${TOKEN_VALUE}\",
+            \"ttl\" : 300,
```

Comment:
Maybe a lower TTL (like 60) is better to prevent stale records.

Reply:
Seems reasonable, updating this.